### ISSUE TITLE
Solidify Rules on Pinging Moderators

### ISSUE THREAD

**Issue by machavity** [SO-Close-Vote-Reviewers/room-meeting-topics#13]
So we already have a general rule about not pinging mods, but we probably should expand that somewhat, since there are some grey areas that people get confused about. I would propose we adopt these three simple rules about pinging mods

1. Pings as part of a discussion you are actively having with a moderator are acceptable
2. Pings about things that can be flagged are unacceptable. If there is an issue needing moderator attention for a comment/question/answer, that's what flags are for, and you will likely get a faster response from the moderator pool as opposed to pinging one specific mod
3. There are some edge cases ([example](https://meta.stackoverflow.com/questions/332808/how-do-you-flag-an-offensive-username-when-the-user-has-no-posts#comment386008_332826)) where a flag is not possible, but still needs moderator attention. In these limited circumstances, it is acceptable to ping a moderator. If you are unsure if it is an edge case, please ask first.

Pings to a moderator that break these rules may be dealt with by kicks and/or removal of the offending messages.

**Comment by Aralun** [comment]
I agree with the intent but couldn't this be simplified?

> Consider moderators in the room as being on leisure time. Do not ping them for issues related to Stack Overflow moderation or support, use flags like any other SO user.

**Comment by machavity** [comment]
How about this?

1. Pinging mods engaged in general conversation is acceptable
2. Pinging mods for anything that could be handled with a flag is not acceptable
3. Ask the room about how to handle non-flag moderation problems

**Comment by makyen** [comment]
Prior to wordsmithing to get something succinct we can put in the FAQ, I think we might benefit by making sure we're all on the same page and also have something more detailed, so we're sure we're not leaving out anything important from the more succinct version.

Pinging can be interpreted in a couple of different ways. As we all know, a "ping" occurs when either using `@username`, or replying to an existing message. Unless we are specific, people will, and have, interpreted it both ways with respect to this issue. Very broadly, I think what we're trying convey is: "Don't drag a mod into talking about specific moderation issues."

More specifically, I think what we're trying to get across is:
* it's fine to:
  1. Engage mods in conversation (using "reply", which causes pings, or explicitly `@[mod]`) about general topics, just like any SO user. However, don't over do it, as Mods always have other things which they *could* be doing on SO, but that doesn't mean they're not human and uninterested in other things (or, [so we think :-)](https://www.xkcd.com/1530/) ).
  2. Use reply/ping to *continue* a conversation begun by the Mod, or willingly joined by the Mod, about moderation topics, including things which would normally be handled with a flag (e.g. Mod comes into SOCVR asking for help checking for plagiarism).
* It's not acceptable to:
  1. Change the topic of a conversation with a mod to a matter which would normally be handled by a flag, when the issue is not *directly* related to the current conversation.
  2. Ping a Mod to initiate a conversation about something which can be handled by a flag or meta post.

In other words, it's not acceptable to impose upon the mod by asking them to deal directly with things which should be handled with flags (or meta posts), just because the mod is currently in, or has been in, SOCVR.

There also appears to be a disparity in style as to how to express this in the FAQ: either having a general statement of purpose, or a list of specific things. Both approaches have their benefits and drawbacks. A more general statement allows us to be less specific, with the implicit expectation that people should use their own judgement. With a list, people will often start thinking about corner cases. In both cases, we should probably have something like: "If in doubt, ask the room what the appropriate thing to do is."
